src/azul/indexer/document.py

```diff
@@ -234,6 +234,7 @@ class FieldType(Generic[N, T], metaclass=ABCMeta):
     shadowed: bool = False
     es_sort_mode: str = 'min'
     allow_sorting_by_empty_lists: bool = True
+    filter_operators: Tuple[str] = ('is',)
```

Comment:
changed from frozenset to tuple to ensure deterministic spec generation.

Reply:
include `within` as a default

Reply:
We can use the first type argument (N) to derive the JSON schema type that most closely resembles the field type.